fs/copy_test.ts

```diff
@@ -82,25 +82,16 @@ testCopy(
 
     const srcContent = new TextDecoder().decode(await Deno.readFile(srcFile));
 
-    assertEquals(
-      await exists(srcFile),
-      true,
-      `source should exist before copy`,
-    );
-    assertEquals(
-      await exists(destFile),
-      false,
+    assert(await Deno.lstat(srcFile), "source should exist before copy");
```

Comment:
It seems there's no unit test for this? I think all of these will throw with an error that won't be as descriptive?

Reply:
That is true. As an alternative, perhaps we could exclusively use `exists` within tests. What do you think?

Reply:
Oh, I didn't pay enough attention to the code to notice this was a test (did a very fast drive by). Yeah, using it in the tests seems like a way to make them more descriptive, or use some local `exists` function to the module.

I'm going to undo my "request changes" because this is just test code.

Reply:
Perhaps, just having a clear `msg` defined within assertion tests will suffice...